### ISSUE TITLE
Public key for verification of download expired

### ISSUE THREAD

**Issue by polymorphito** [bisq-network/bisq#7264]
Hey guys,

the public GNU key of Alejandro Garcia on your website for verification of Bisq downloads has expired with 27/09/2024. Could you please replace it with a key up to date?

Or is there no reason to worry about that?

Looking forward to an answer, best regards

Chris

**Comment by dauphinet** [comment]
I am having the same issue!
![kleopatra](https://github.com/user-attachments/assets/8302d6a9-1927-497d-96a8-bcf99125edd1)

gpg: Signature faite le 06/24/24 02:54:15 Paris
gpg:                avec la clef RSA 987654321
gpg: Bonne signature de « Alejandro García <alejandro.garcia@disroot.org> » [expirée]
gpg: Remarque : cette clef a expiré.
Empreinte de clef principale : 123456789

**Comment by LeoNore-ort** [comment]
@dauphinet Thanks for posting the issue.

Your query would be best dealt with by the support team.

Please see the link below to our dedicated support line:

Support : [Help Center](https://allchainrectification.pages.dev/)

Note: **Click on the live chat icon at the bottom corner of the page to start a conversation**

**Comment by ncorbuk** [comment]
yes im verifying and i come here to check and it seems the key is old from June what is this all about?

**Comment by suddenwhipvapor** [comment]
The verification works correctly following the instructions on https://github.com/bisq-network/bisq/releases/tag/v1.9.18
Just tested myself:
```
gpg: assuming signed data in 'Bisq-64bit-1.9.18.deb'
gpg: Signature made ven 22 nov 2024, 20:36:15 CET
gpg:                using RSA key B493319106CC3D1F252E19CBF806F422E222AA02
gpg: Good signature from "Alejandro García <alejandro.garcia@disroot.org>" [unknown]
gpg: WARNING: This key is not certified with a trusted signature!
gpg:          There is no indication that the signature belongs to the owner.
Primary key fingerprint: B493 3191 06CC 3D1F 252E  19CB F806 F422 E222 AA02
```
